inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/UserServiceImpl.java

```diff
@@ -90,6 +97,19 @@ public boolean create(UserInfo userInfo) {
         entity.setDueDate(getOverDueDate(userInfo.getValidDays()));
         entity.setCreateBy(LoginUserUtils.getLoginUserDetail().getUserName());
         entity.setName(username);
+        entity.setSecretKey(RandomStringUtils.randomAlphanumeric(8));
+        try {
+            Map<String, String> keyPairs = RSAUtils.generateRSAKeyPairs();
+            String publicKey = keyPairs.get(RSAUtils.PUBLIC_KEY);
+            String privateKey = keyPairs.get(RSAUtils.PRIVATE_KEY);
+            entity.setPublicKey(publicKey);
+            entity.setPrivateKey(privateKey);
```

Comment:
In my opinion, SecretKey should be encrypted  once again after persisted in db if possible

Reply:
Agreed. working on it